src/components/testCaseLanding/common/TestCaseListSideBarNav.test.tsx

```diff
@@ -22,13 +22,10 @@ const groups: Group[] = [
 ];
 jest.mock("@madie/madie-util", () => ({
   useFeatureFlags: jest.fn(() => {
-    return {
-      includeSDEValues: true,
-    };
+    return {};
```

Comment:
this is not required if you remove its usage in component 
useFeatureFlags `const featureFlags = useFeatureFlags();`